class/bike/dead/DeadRider.js

```diff
@@ -92,18 +92,24 @@ export class DeadRider {
         // drawer.stroke();
     }
 
-    update() {
+    fixedUpdate() {
         for (let i = this.joints.length - 1; i >= 0; i--) {
-            this.joints[i].update();
+            this.joints[i].fixedUpdate();
         }
         for (let i = this.points.length - 1; i >= 0; i--) {
-            this.points[i].update();
+            this.points[i].fixedUpdate();
         }
     }
 
-    pull(upperForce, lowerForce) {
-        upperForce.selfScale(0.7);
-        lowerForce.selfScale(0.7);
+    update(progress) {
+        for (let mass of this.points) {
+            mass.update(progress);
+        }
+    }
+
+    setVelocity(upperVel, lowerVel) {
```

Comment:
I renamed this function, as well as a few variables within it. No functionality was changed